docs/sql-pyspark-pandas-with-arrow.md

```diff
@@ -22,7 +22,7 @@ license: |
 * Table of contents
 {:toc}
 
-## Apache Arrow in Spark
+## Apache Arrow in PySpark
```

Comment:
I realised that other Arrow optimization might likely be placed in other places .. :).

Reply:
Make sense. The filename is sql-pyspark...